vm-setup/roles/packages_installation/defaults/main.yml

```diff
@@ -24,6 +24,10 @@ packages:
      - libpolkit-agent-1-0
      - libpolkit-backend-1-0
      - libpolkit-gobject-1-0
+    pip:
+     - setuptools 
+     - libvirt-python
+     - lxml
```

Comment:
Can you add a comment about those packages ? How they are used and why they need to stay in python2 ?

Reply:
Done